infra/src/main/kotlin/fi/fta/geoviite/api/frameconverter/v1/FrameConverterControllerV1.kt

```diff
@@ -27,6 +29,7 @@ class FrameConverterApiObjectMapperV1 {
     }
 }
 
+@PreAuthorize(AUTH_API_FRAME_CONVERTER)
```

Comment:
Varsinainen privilege-tarkistus on toistaiseksi koko luokalle (jokaiselle funkkarille voisi tämän toki laittaa myös erikseen kuten aiemmissa kontrollereissa)

Reply:
Tämä lienee ihan ok näin. Muutetaan sitten jos joskus tulee eri oikeuksien takana olevia rajapintoja